external-crates/move/crates/move-analyzer/src/completions/name_chain.rs

```diff
@@ -151,8 +151,11 @@ pub fn name_chain_completions(
                         mod_ident.module.value().as_str(),
                         CompletionItemKind::MODULE,
                     );
-                    let auto_import_text =
-                        format!("use {}::{}", mod_ident.address, mod_ident.module);
+                    let auto_import_text = format!(
+                        "use {}::{}",
+                        addr_to_ide_string(&mod_ident.address),
```

Comment:
All calls to `addr_to_ide_string` are there to make sure that even in case of name conflicts we print a correct package name (or address) and not the combination of the two. This applies both to auto-imports and auto-fixes